ldm/models/diffusion/shared_invokeai_diffusion.py

```diff
@@ -93,6 +98,21 @@ def override_cross_attention(self, conditioning: ExtraConditioningInfo, step_cou
         the previous attention processor is returned so that the caller can restore it later.
         """
         self.conditioning = conditioning
+
+        # If other modules do not want cross_attention_control then we should bypass setting up Context
+        old_attn_processors = None
```

Comment:
this logic seems a little hairy. have you tested `.swap()` with `withLora()`? they should still play nice together